index.js

```diff
@@ -0,0 +1,31 @@
+var calculator = new Vue({
+    el: '#calculator',
```

Comment:
Never worked with Vue before, what does this line mean?
el: '#calculator'

is that just some identifier for the element?